internal_transfers/actions/src/simulate/interface.ts

```diff
@@ -0,0 +1,44 @@
+import { EventLog } from "../models";
+
+export interface SimulationParams {
+  // Ethereum address transaction should be sent to.
+  contractAddress: string;
+  // Ethereum address of transaction sender.
+  sender: string;
+  // 0x-prefixed string representing transaction data to simulate.
+  callData: string;
+  // Amount of Ether to send along with transaction.
+  value: string;
+  // Block at which simulation should be made.
+  blockNumber?: number;
```

Comment:
Out of curiosity: Why can a `blockNumber` be optional?

Reply:
This is a tenderly thing for the most part, when the block number is not specified the simulation is made on the current or latest block.

Reply:
```suggestion
  // use "latest" when undefined.
  blockNumber?: number;
```